docs/people-and-groups/saml-okta.md

```diff
@@ -6,26 +6,64 @@ title: SAML with Okta
+
+### Configure Okta SAML settings
+
+To configure Okta app integration with Metabase, you'll need to use the information in Metabase **Admin panel** > **Authentication** > **SAML**.
+
+#### General settings
+
+| Okta SAML                       | Metabase SAML                                                                                                                               |
+| ------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Single sign-on URL**          | **URL the IdP should redirect to**. This is your Metabase [Site URL][site-url] -- it should start with `https://` and end with `/auth/sso`. |
+| **Audience URI (SP Entity ID)** | **SAML Application Name** ("Metabase" by default)                                                                                           |
+
+#### Attribute statements
+
+Metabase requires you to set some attributes in Okta:
+
+- email address
+- first name (given name)
+- last name (surname)
+
+These attributes will be passed from Okta to Metabase during authentication to automatically log people into their Metabase accounts.
+
+In **Attribute statements (optional)** section of the Okta application SAML setting, create attribute statements for the email, first name, and last name.
 
 | Name                                                                 | Value          |
 | -------------------------------------------------------------------- | -------------- |
-| `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname`    | user.firstName |
 | `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress` | user.email     |
+| `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname`    | user.firstName |
 | `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname`      | user.lastName  |
 
-**End-users should not be able to edit the email address attribute**. Your IdP will pass the email address attribute to Metabase in order to log people into their Metabase accounts (or to create an account on the first login). If a person can change the email address attribute, they'll potentially be able to access Metabase accounts other than their own.
+The names of attribute statement in Okta should match the attribute names in Metabase. If you use non-default attribute names in Okta app configuration, you will also need to change the names for the attribute fields in Metabase in **Admin panel** > **Authentication** > **SAML**
+
+> **End-users should not be able to edit the email address attribute**. Your IdP will pass the email address attribute to Metabase in order to log people into their Metabase accounts (or to create an account on the first login). If a person can change the email address attribute, they'll potentially be able to access Metabase accounts other than their own.
```

Comment:
```suggestion
> **Make sure that people cannot edit their email address attribute**. To log people in to your Metabase (or to create a Metabase account on first login), your IdP will pass the email address attribute to Metabase. If a person can change the email address attribute, they'll potentially be able to access Metabase accounts other than their own.
```

Also: how should an admin make sure that people can't edit their email address attribute? Is there an explicit action the admin needs to take to ensure that the email attribute is uneditable by account holders?

Reply:
@jeff-bruemmer To be honest, I am not sure, I kept this as-is from the previous version of the docs. We say this in all SAML docs but don't  explain how to make this happen. I think it's a property of the user profile attribute in the IdP